explainer.md

```diff
@@ -163,41 +168,46 @@ for await (const datagram of transport.receiveDatagrams()) {
 const mediaSource = new MediaSource();
 await new Promise(resolve => mediaSource.addEventListener('sourceopen', () => resolve(), {once: true}));
 const sourceBuffer = mediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
-const transport = new Http3Transport("https://example.com/video");
-for await (const stream of transport.receiveStreams()) {
-  sourceBuffer.appendBuffer(await stream.arrayBuffer());
-  await new Promise(resolve => sourceBuffer.addEventListener('update', () => resolve(), {once: true}));
+const transport = new WebTransport('https://example.com/video');
+for await (const receiveStream of transport.incomingUnidirectionalStreams) {
+  for await (const buffer of receiveStream.readable) {
+    sourceBuffer.appendBuffer(buffer);
+  }
+  await new Promise(resolve => sourceBuffer.addEventListener('update', resolve, {once: true}));
 }
 ```
 
 ## Detailed design discussion
 
-Any WebTransport can provide any of the following capabilities (mixins):
+WebTransport supports multiple protocols, each of which provide some of the
+following capabilities.
 
-- Unidirectional streams are indefintely long streams of bytes in one direction 
+- Unidirectional streams are indefintely long streams of bytes in one direction
   with back pressure applied
   to the sender when either the receiver can't read quickly enough or when
   constrained by network capacity/congestions.  Useful for sending messages that
   do not expect a response.  In-order, reliable messaging can be achieved by
   sending many messages in a single stream. Out-of-order messaging can be achieved
   by sending one message per stream.
 
-- Bidirectional streams are like unidirectional streams, but in two directions.  
+- Bidirectional streams are like unidirectional streams, but in two directions.
   They are useful for sending messages that expect a response.
 
-- Datagrams are small, out-of-order, unreliable messages.  They are useful for 
+- Datagrams are small, out-of-order, unreliable messages.  They are useful for
   sending messages with less API complexity
   and less network overhead than streams.
 
-A QuicTransport is a WebTransport that maps directly to QUIC streams and
-datagrams, which makes it easy to connect to servers that speak QUIC with
-minimum overhead.  It supports all of these capabilities.
+[QuicTransport](https://tools.ietf.org/html/draft-vvv-webtransport-quic)
+is a WebTransport that maps directly to QUIC streams and datagrams, which makes
```

Comment:
I found "a WebTransport" to be confusing here. Can we say "a WebTransport protocol"?

Reply:
Done.